src/rules/selector-attribute-quotes/index.js

```diff
@@ -0,0 +1,70 @@
+import {
+  isStandardSyntaxRule,
+  isStandardSyntaxSelector,
+  parseSelector,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "selector-attribute-quotes"
+
+export const messages = ruleMessages(ruleName, {
+  expectedQuotes: "Expected quotes",
+  rejectedQuotes: "Unexpected quotes",
+})
+
+function isQuotesExist(str) {
```

Comment:
> i think it is bug in postcss-selector-parser, you are create PR?

I'm afraid I'm going away until Monday, so I doubt I'll get the chance to look into it until next week. Either of you two, feel free to resolve this in my absence though :)